api/lib/domain/events/handle-complementary-certifications-scoring.js

```diff
@@ -126,6 +134,10 @@ async function _saveResult({
       acquired,
     }),
   );
+
+  if (acquired && lowerLevelComplementaryCertificationBadgeAcquired) {
```

Comment:
```suggestion
  if (lowerLevelComplementaryCertificationBadgeAcquired) {
```
Ça me semble être la même chose. Ça me choque pas en l'état non plut 🤔

Reply:
Oui tu as raison puisque qu'on check déjà l'obtention dans `getComplementaryCertificationResultInformation` et ça allège le code.